test/test_trigger.py

```diff
@@ -5,9 +5,9 @@
 
 
 class FakeCircleCIClient(object):
-    def __init__(self, api_key):
+    def __init__(self, api_token):
         self._builds_triggered = []
-        self._api_key = api_key
+        self.api_token = api_token
```

Comment:
This change needed to be made because the real circleclient exposes an `api_token` param which I now make use of in cinderblock